tensorflow_addons/layers/normalizations_test.py

```diff
@@ -331,5 +332,73 @@ def test_groupnorm_convnet_no_center_no_scale(self):
         )
 
 
+@test_utils.run_all_in_graph_and_eager_modes
```

Comment:
We're trying to get rid of the `run_all_in_graph_and_eager_mode` which is a private decorator. You can look at #1328 for the instructions on how to update. It's fairly easy and you can ping me if you have issues.

Reply:
Will do that. Thanks